src/allmydata/web/status.py

```diff
@@ -673,90 +673,129 @@ def render_progress(self, ctx, data):
     def render_status(self, ctx, data):
         return data.get_status()
 
-class RetrieveStatusPage(rend.Page, RateAndTimeMixin):
-    docFactory = getxmlfile("retrieve-status.xhtml")
 
-    def __init__(self, data):
-        rend.Page.__init__(self, data)
-        self.retrieve_status = data
 
-    def render_started(self, ctx, data):
-        started_s = render_time(data.get_started())
-        return started_s
+# Renders "/status/retrieve-%d".
+class RetrieveStatusPage(MultiFormatResource):
 
-    def render_si(self, ctx, data):
-        si_s = base32.b2a_or_none(data.get_storage_index())
+    def __init__(self, retrieve_status):
+        """
+        :retrieve_status retrieve.RetrieveStatus: stats provider.
```

Comment:
needs `param` (and the type is first) .. `:param <optional type> arg_name: the description`

Reply:
Made this change in 812f039.

Reply:
Made another change in 9d5d338.  I missed the `:param <type> <name>` format, but then so did `pydocstyle`...